.env

```diff
@@ -0,0 +1,6 @@
+# Comunication With Server
+NEXT_PUBLIC_API_URL=https://ez2live-dev-c781db305a07.herokuapp.com/v1
+
+# DATABASE
+MONGODB_URL=mongodb+srv://dbuser:0KEOy46rIJJ4Pq3D@dev-cluster.mzkybvz.mongodb.net/?retryWrites=true&w=majority
```

Comment:
Please remove this. MongoDB URL is not need in the front-end. Also the password is being leaked.